apps/etterlatte-saksbehandling-ui/client/src/components/nyoppgavebenk/ToggleMinOppgaveliste.tsx

```diff
@@ -63,6 +63,10 @@ export const ToggleMinOppgaveliste = () => {
       const index = oppdatertOppgaveState.findIndex((o) => o.id === id)
       oppdatertOppgaveState[index].saksbehandler = saksbehandler
       oppdatertOppgaveState[index].status = 'UNDER_BEHANDLING'
+      let gjeldendeVersjon = oppdatertOppgaveState[index].versjon
+      if (gjeldendeVersjon) {
+        oppdatertOppgaveState[index].versjon = ++gjeldendeVersjon
+      }
```

Comment:
Dette er en forholdsvis pragmatisk approach. Vi _kan_ implementere at versjon oppdateres på bakgrunn av retur fra tjenestene i `behandling`

Reply:
En vellykket update til Gosys medfører at versjonen der oppdateres, og det reflekteres i responsen. Versjonen økes da med 1. Så er det opp til oss om vi ønsker å mappe versjonsnummeret fra responsen eller bare bumper det i UI ved vellykket kall. Jeg begynte litt på det å oppdatere på bakgrunn av respons, men det blir en ganske stor endring.

Reply:
Hvis vi ikke bumper (på et eller annet vis), så vil en påfølgende oppdatering til Gosys feile da vi sender med versjonsnummer lavere enn gjeldende i Gosys, med mindre man har hentet oppgaven(e) på nytt i mellomtiden.

Reply:
presisering: **ikke** _just in case_. Gosys bumper alltid, men vi har ikke tatt høyde for det.